build/build.go

```diff
@@ -290,7 +290,7 @@ func parseAndAugment(pkg *build.Package, isTest bool, fileSet *token.FileSet) ([
 			switch d := decl.(type) {
 			case *ast.FuncDecl:
 				if replacedDeclNames[funcName(d)] {
-					d.Name = ast.NewIdent("_")
+					d.Name.Name = "_gopherjs_overriden_" + d.Name.String()
```

Comment:
You've addressed point 2 but still not 1. Did you miss it, or are you not making the change because you disagree it's an improvement?